workshop.wwwapi/Models/Patient.cs

```diff
@@ -4,9 +4,42 @@
 namespace workshop.wwwapi.Models
 {
     //TODO: decorate class/columns accordingly    
+    [Table("patients")]
     public class Patient
-    {        
-        public int Id { get; set; }        
+    {
+        [Column("id")]
+        public int Id { get; set; }
+        [Column("fullname")]
         public string FullName { get; set; }
+
+        public ICollection<Appointment> Appointments { get; set; }
+    }
+
+    public class AppointmentPatientDTO
```

Comment:
please move DTOs into separate file and folder; they are not related to the models layer and are related to the api / endpoints layer